### DISCUSSION TITLE
SQLAlchemy-2.0.4 bulk_query return null but write failed

### DISCUSSION THREAD

**Discussion by WhereTo** [sqlalchemy/sqlalchemy#9441] [Usage Questions]
### Describe the use case

SQLAlchemy-2.0.4 bulk_query return null but write failed

### Databases / Backends / Drivers targeted

mysql

### Example Use

bulk_query（inserSQL）；return none but check the mysql， there are not new record

**Comment by CaselIT** [comment]
Hi

There is nothing actionable with the provided information. Can you provide additional details?
